start_esp/nginx-auto.conf.template

```diff
@@ -50,6 +50,7 @@ http {
   server_tokens off;
   client_max_body_size ${client_max_body_size};
   client_body_buffer_size ${client_body_buffer_size};
+  large_client_header_buffers ${large_client_header_buffers};
```

Comment:
Prefer not to add this if the flag is not specified.   It will not impact the users not using the flag.